runtime/parachains/src/paras.rs

```diff
@@ -1005,11 +1005,16 @@ impl<T: Config> Pallet<T> {
 
 				let (reads, writes) = Self::increase_code_ref(&new_code_hash, &new_code);
 				FutureCodeHash::<T>::insert(&id, new_code_hash);
-				T::DbWeight::get().reads_writes(2 + reads, 3 + writes)
+				T::DbWeight::get().reads_writes(3 + reads, 5 + writes)
```

Comment:
How can I verify these?

Reply:
Example: https://docs.rs/frame-support/latest/frame_support/storage/trait.StorageValue.html#tymethod.mutate

public mutate: https://github.com/paritytech/substrate/blob/0e6cc5668d9ee8d852a3aa3f85a2ab5fcb4c75a1/frame/support/src/storage/types/value.rs#L134-L136

which calls private mutate: https://github.com/paritytech/substrate/blob/master/frame/support/src/storage/generator/value.rs#L104-L106

which calls try_mutate: https://github.com/paritytech/substrate/blob/0e6cc5668d9ee8d852a3aa3f85a2ab5fcb4c75a1/frame/support/src/storage/generator/value.rs#L108-L119

which calls get https://github.com/paritytech/substrate/blob/0e6cc5668d9ee8d852a3aa3f85a2ab5fcb4c75a1/frame/support/src/storage/generator/value.rs#L109 and calls either put: https://github.com/paritytech/substrate/blob/0e6cc5668d9ee8d852a3aa3f85a2ab5fcb4c75a1/frame/support/src/storage/generator/value.rs#L114 or kill https://github.com/paritytech/substrate/blob/0e6cc5668d9ee8d852a3aa3f85a2ab5fcb4c75a1/frame/support/src/storage/generator/value.rs#L115

The `get` counts as a read. The `kill` or `put` counts as a write

Reply:
Thank you for exercising this all the way through :)